satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -338,163 +407,147 @@ datasets:
 
   product_quality_oca:
     name: product_quality_oca
     file_type: nc_fci_oca
     file_key: product_quality
-    long_name: product_quality_index
+    standard_name: product_quality
 
   product_completeness_oca:
     name: product_completeness_oca
     file_type: nc_fci_oca
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_oca:
     name: product_timeliness_oca
     file_type: nc_fci_oca
     file_key: product_timeliness
-    long_name: product_timeliness_index
+    standard_name: product_timeliness
 
   # FIR
   fire_probability:
     name: fire_probability
     resolution: 2000
     file_type: nc_fci_fir
     file_key: fire_probability
+    standard_name: fire_probability
 
   fire_result:
     name: fire_result
     resolution: 2000
     file_type: nc_fci_fir
     file_key: fire_result
+    standard_name: active_fire_classification
+    fill_value: -127
+    import_enum_information: True
 
   product_quality_fir:
     name: product_quality_fir
     file_type: nc_fci_fir
     file_key: product_quality
-    long_name: product_quality_index
+    standard_name: product_quality
 
   product_completeness_fir:
     name: product_completeness_fir
     file_type: nc_fci_fir
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_fir:
     name: product_timeliness_fir
     file_type: nc_fci_fir
     file_key: product_timeliness
-    long_name: product_timeliness_index
+    standard_name: product_timeliness
 
   # OLR
   olr:
     name: olr
     resolution: 2000
     file_type: nc_fci_olr
     file_key: olr_value
-    long_name: outgoing_longwave_radiation
+    standard_name: outgoing_longwave_radiation
 
   cloud_type_olr:
     name: cloud_type_olr
     resolution: 2000
     file_type: nc_fci_olr
     file_key: cloud_type
-    long_name: cloud_type_olr
+    standard_name: cloud_type_classification
+    fill_value: -127
+    import_enum_information: True
 
   quality_overall_processing_olr:
     name: quality_overall_processing_olr
     resolution: 2000
     file_type: nc_fci_olr
     file_key: quality_overall_processing
-    long_name: quality_index
+    standard_name: quality_flag
+    fill_value: -127
+    import_enum_information: True
 
   product_quality_olr:
     name: product_quality_olr
     file_type: nc_fci_olr
     file_key: product_quality
-    long_name: product_quality_index
+    standard_name: product_quality
 
   product_completeness_olr:
     name: product_completeness_olr
     file_type: nc_fci_olr
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_olr:
     name: product_timeliness_olr
     file_type: nc_fci_olr
     file_key: product_timeliness
-    long_name: product_timeliness_index
+    standard_name: product_timeliness
 
   # CRM
```

Comment:
Any other datasets than CRM & ASR for which you want to overwrite the long_names? Because otherwise I don't add this entry, meaning the netCDF one is used.

Reply:
Do you want to keep the "(7 days average)" in the long name for CRM, although for FCI it's configurable?

Reply:
I would say "(temporal average)"

Reply:
done

Reply:
Seems like the long names were removed again with this commit: https://github.com/pytroll/satpy/pull/2665/commits/1c02d1c7bf310683c44e5fa90acc864db671fc7d

Furthermore this point was never fixed I think: "2. Set the units to %, set to "none" in the netCDF file."

Reply:
Fixed in commit a02511f0a8ea1e5144223fc1d4e2457f1cbc3ff3